Dockerfile

```diff
@@ -27,6 +31,22 @@ RUN curl -O https://download.clojure.org/install/linux-install-${CLOJURE_VERSION
       && chmod +x linux-install-${CLOJURE_VERSION}.sh \
       && ./linux-install-${CLOJURE_VERSION}.sh
 
+# Install Planck so we can run our tests in self-hosted mode.
+
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
+RUN apt-get install -qq -y \
+      libjavascriptcoregtk-4.0 \
+      libglib2.0-dev \
+      libzip-dev \
+      libcurl4-gnutls-dev \
+      libicu-dev
+
+RUN git clone https://github.com/planck-repl/planck.git
+RUN cd planck && script/build --fast \
+      && script/install \
+      && planck -h \
+      && cd ..
+
```

Comment:
My understand of Dockerfile best practices is to combine `RUN` commands to the extent practical / reasonable (see "Minimize the number of layers" at https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). The docs imply the performance justification for that practice isn't a concern any longer, but I've found that the pattern you generally see in extant Dockerfiles is to have `apt-get` commands combined into a single one, and to have the `git clone ; cd <project> ; build ...` commands combined into a single `RUN`...

Reply:
I see! I'll make that change.

Reply:
Captured that as #139.